tests/test_sft_trainer.py

```diff
@@ -431,17 +429,9 @@ def _get_adapter_config(dir_path):
         return json.load(f)
 
 
-def _validate_adapter_config(adapter_config, peft_type, tuning_config):
+def _validate_adapter_config(adapter_config, peft_type):
     assert adapter_config.get("task_type") == "CAUSAL_LM"
     assert adapter_config.get("peft_type") == peft_type
-    assert (
-        (
```

Comment:
this test is checking if adapters produced after tuning has the correct fields. I think its worthwhile to check if `adapter_config.get("tokenizer_name_or_path") == MODEL_ARGS.tokenizer_name_or_path`

def _validate_adapter_config(adapter_config, peft_type, MODEL_ARGS.tokenizer_name_or_path)